src/main/java/dal/TicketManager.java

```diff
@@ -0,0 +1,23 @@
+package dal;
+
+import model.VehicleType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TicketManager {
```

Comment:
Ticket Manager maybe makes sense in a realistic environment but in our case `Employee` should be responsible for charging parking and refilling tanks.